src/common/components/select-repository-list/index.js

```diff
@@ -149,13 +168,13 @@ export class SelectRepositoryListComponent extends Component {
             <div className={ styles.arrow } />
           }
           <div className={ styles.summary }>
-            <strong>
-              { selectedRepositories.length } selected
-            </strong>
-            {' '}
-            (<Button appearance={ 'link' } onClick={this.onHelpClick.bind(this)}>
-              { this.state.showMissingReposInfo ? 'Return to repos list' : 'Anything missing?' }
-            </Button>)
+            { this.renderRepoAmount() }
+            <div>
```

Comment:
But now with `div` removed from `renderRepoAmount` they should be inline (as they are just inline elements), right?